lib/src/components/buttons/help_option_button.dart

```diff
@@ -22,34 +19,43 @@ class HelpOptionButton extends StatelessWidget {
   /// hovering over it.
   final Color? hoverColor;
 
-  static const _defaultHeight = 53.0;
+  /// Sets huge modifcation
+  final bool huge;
```

Comment:
Not sure about this `huge` toggle.

Instead of switching between predefined sizes, maybe we should parametrize the class?
Allow setting custom spacings etc?